dist/index.html

```diff
@@ -1,201 +1,53 @@
-          </div>
-        </section>
-        <section class="infoContainer-cardContainer-card-vertical">
-          <div class="card-history-sleep">
-            <p class="thisWeek">Hours of sleep this week</p>
-            <div class="sidebar-header-line"></div>
-            <ul class="card-vertical-list" id="sleepThisWeek">
-              <!-- Sleep week list goes here -->
-            </ul>
-          </div>
-          <div class="card-history-sleep">
-            <p class="historicalWeek"></p>
-            <div class="sidebar-header-line"></div>
-            <ul class="card-vertical-list" id="sleepEarlierWeek">
-              <!-- Sleep week list goes here -->
-            </ul>
-          </div>
-        </section>
-      </section>
-    </section>
-    <section class="main-sidebar-body">
-      <section class="body-sidebar-sidebarContainer">
-        <h2 class="sidebar-header-name" id="sidebarName">FRIENDLY STEP CHALLENGE!</h2>
-        <div class="sidebar-header-line"></div>
-        <img src="./images/person walking on path.jpg" class="sidebar-header-userImage"></img>
-        <div class="sidebar-header-line"></div>
-        <p class="sidebar-header-userInfo" id="bigWinner"></p>
-        <div class="sidebar-header-line"></div>
-        <p class="sidebar-header-userInfo" id="stepGoalCard"></p>
-        <p class="sidebar-header-userInfo" id="avStepGoalCard">
-        <p>
-        <section class="sidebar-body-friendContainer">
-          <section class="sidebar-friendContainer-listItems">
-            <p class="thisWeek">Rank this week</p>
-            <ul class="card-vertical-list" id="friendChallengeListToday">
-              <!-- friend list goes here -->
-            </ul>
-            <div class="sidebar-header-line"></div>
-            <p class="historicalWeek">- Historical Rank</p>
-            <ul class="card-vertical-list" id="friendChallengeListHistory">
-              <!-- friend list goes here -->
-            </ul>
-            <div class="sidebar-header-line"></div>
-          </section>
-          <p class="thisWeek">You had 3 DAY STEP STREAKS on these days:</p>
-          <ul class="card-vertical-list" id="streakList">
-            <!-- friend list goes here -->
-          </ul>
-        </section>
+      <section class="sidebar">
+        <h2>Friendly Step Challenge!</h2>
+        <img src="./images/person-walking-on-path.jpg" alt="A picture of a pair of legs, walking forward down a sidewalk" class="sidebar-image">
+        <p id="bigWinner"></p>
+        <p>Rank this week</p>
+        <ul class="card-list" id="friendChallengeListToday"></ul>
+        <p>Historical Rank</p>
+        <ul class="card-list" id="friendChallengeListHistory"></ul>
+        <p>You had 3-Day Step Streaks on these days:</p>
+        <ul class="card-list" id="streakList"></ul>
       </section>
-    </section>
-  </section>
-
-
-  <script src="bundle.js"></script>
-</body>
-
+    </main>
+    <script src="bundle.js"></script>
```

Comment:
Wow, I knew there was extra fluff but you really thinned this down! I love the explicit tags too!